test/test_python.py

```diff
@@ -530,3 +530,12 @@ def test_runpythonasync_exception_after_import(selenium_standalone):
         assert "ZeroDivisionError" in str(e)
     else:
         assert False
+
+
+def test_cprofile(selenium_standalone):
```

Comment:
So the following still fails, 
```
XFAIL test/test_python.py::test_cpython_core[test_cprofile-firefox]
  reason: known failure with code "_lsprof"
```
but I assume with a different error message? The previous one was,
```
== CPython 3.7.0 (default, Oct 15 2018, 10:45:15) [Clang 6.0.1 ]
== Emscripten-1.0-x86-JS-32bit little-endian
== cwd: /tmp/test_python_42
== CPU count: 4
== encodings: locale=UTF-8, FS=utf-8
Run tests sequentially
0:00:00 load avg: 0.10 [1/1] test_cprofile
test test_cprofile crashed -- Traceback (most recent call last):
  File "/lib/python3.7/test/libregrtest/runtest.py", line 163, in runtest_inner
    the_module = importlib.import_module(abstest)
  File "/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/lib/python3.7/test/test_cprofile.py", line 7, in <module>
    import cProfile
  File "/lib/python3.7/cProfile.py", line 9, in <module>
    import _lsprof
ModuleNotFoundError: No module named '_lsprof'

test_cprofile failed
```

Reply:
Actually, I was wondering it did fail. This can be checked with,
```
pytest test/test_python.py::test_cpython_core -k cprofile --run-xfail 
```
(XFAIL test are currently not run by default).

Reply:
Ah, I've updated python_tests.txt now to expect a working test there.

Reply:
If the code Cpython test works, do we need this one? This one is slow as it creates a new selenium instance...

Reply:
Good point.  I suppose not.  Removing...